lib/engine/game/g_18_mag.rb

```diff
@@ -3,6 +3,9 @@
 module Engine
   module Game
     module G18Mag
+      def available_programmed_actions
```

Comment:
I don't think this is the right place for this.

Reply:
No, I'll fix after Toby reviews

Reply:
Kinda confused why 18Mag is in here at all.

Reply:
18mag does actually float corps